guide/14-deep-learning/track_objects_using_SiamMask.ipynb

```diff
@@ -100,7 +100,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Previous:</p><p><strong><em>Step 3</em></strong>: Load the model, which is saved inside 'torch_scripts' folder, in <em>ArcGIS Pro 2.8</em>. To use the model, follow <a href="https://pro.arcgis.com/en/pro-app/latest/help/analysis/image-analyst/object-tracking-in-motion-imagery.htm" target="_blank">Object tracking in motion imagery</a>.</p><p> </p><p>Suggested:</p><p><strong><em>Step 3</em></strong>: Load the model, which is saved inside 'torch_scripts' folder, in <em>ArcGIS Pro 2.8</em>. Follow <a href="https://pro.arcgis.com/en/pro-app/latest/help/analysis/image-analyst/object-tracking-in-motion-imagery.htm" target="_blank">Object tracking in motion imagery</a> to track object.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>Done</p>